lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -692,21 +695,31 @@ defaultTilingOfType(RewriterBase &rewriter, Operation *op,
       // All parallel iterator will be tiled by `32` or `16`. If need
       // specified, please set option `defaultTileSize`, like `matmul:{64,64}`.
       if (iterType == utils::IteratorType::parallel) {
-        Range curDomain = iterationDomain[en];
-        std::optional<int64_t> tripCount = mlir::constantTripCount(
-            curDomain.offset, curDomain.size, curDomain.stride);
-        if (tileDims >= 2 && en > 0) {
+// Use 2D-tile by default.
```

Comment:
It would be better to make this as pass parameter.

Reply:
Good point! Add it to new pass parameter called `default-nd-tile`.